sw/device/tests/clkmgr_smoketest.c

```diff
@@ -35,6 +35,8 @@ static void test_gateable_clocks(const dif_clkmgr_t *clkmgr) {
       CHECK_DIF_OK(
           dif_clkmgr_gateable_clock_get_enabled(clkmgr, clock, &actual_state));
       CHECK(actual_state == expected_state);
+
+      state = actual_state;
```

Comment:
I don't think the `actual_state` variable serves a useful purpose. We could just drop it.

```c
      dif_toggle_t expected_state =
          state == kDifToggleEnabled ? kDifToggleDisabled : kDifToggleEnabled;
      CHECK_DIF_OK(
          dif_clkmgr_gateable_clock_set_enabled(clkmgr, clock, expected_state));
      CHECK_DIF_OK(
          dif_clkmgr_gateable_clock_get_enabled(clkmgr, clock, &state));
      CHECK(state == expected_state);
```